liana-gui/src/app/state/psbts.rs

```diff
@@ -31,7 +31,7 @@ impl PsbtsPanel {
     }
 
     pub fn preselect(&mut self, spend_tx: SpendTx) {
-        let psbt_state = psbt::PsbtState::new(self.wallet.clone(), spend_tx, true);
+        let psbt_state = psbt::PsbtState::new(self.wallet.clone(), spend_tx, true, None);
```

Comment:
I know you addressed the error in the view page but can we not populate the bip21 here?

Reply:
No. But this brings up an interesting point. We dont save BIp21s after we use them and that is a form of information loss. Meaning if you save a send with a bip21. And that bip21 included pj details then that information is lost forever and that spend needs to be a tranditional payment now.